ietf-flexi-grid-path-computation.tree

```diff
@@ -2,10 +2,53 @@ module: ietf-flexi-grid-path-computation
 
   augment /te:tunnels-path-compute/te:input/te:path-compute-info
             /tepc:path-request:
-    +-- fec-type?                identityref
-    +-- termination-type?        identityref
-    +-- bit-stuffing?            boolean
-    +-- wavelength-assignment?   identityref
+    +-- fec-type?                      identityref
+    +-- termination-type?              identityref
+    +-- bit-stuffing?                  boolean
+    +-- wavelength-assignment?         identityref
+    +-- flexi-grid-path-constraints
+       +-- gsnr-margin?   snr
```

Comment:
As discussed, since the gsnr-margin is applicable to both wson and flex grid there is no need for an extra container

Reply:
Fixed in https://github.com/aguoietf/ietf-ccamp-optical-path-computation/pull/18/commits/27352dc7be50c1ccd4f5e7d52b98e3d633f0fca9